{
  try {
    root.delete("unknown file");
    fail();
  }
 catch (  AssertionError e) {
  }
}
